packages/api/tests/models/test-rules-model.js

```diff
@@ -273,3 +273,17 @@ test.serial('Creating a kinesis rule where an event source mapping already exist
     lambdaStub.reset();
   }
 });
+
+// This test currently fails because queueName is not an acceptable top
+// level parameter in the rule schema. When the work is done to change
+// the dashboard, that should be updated.
+test.skip('Creating a rule with a queueName parameter', async (t) => {
+  const ruleItem = Object.assign({ queueName: 'testQueue' }, onetimeRule);
```

Comment:
I would generally put the object for the properties you want to be added as the second parameter, just to be sure that if a `queueName` property exists on `onetimeRule`, it does not override what you're expecting to see